test/GetResourceHeadersTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Handler\MockHandler;
+use GuzzleHttp\HandlerStack;
+use GuzzleHttp\Psr7\Response;
+use Islandora\Chullo\Chullo;
+
+class GetResourceHeadersTest extends \PHPUnit_Framework_TestCase {
+
+    /**
+     * @covers  Islandora\Fedora\Chullo::getResourceHeaders
+     * @uses    GuzzleHttp\Client
+     */
+    public function testReturnsHeadersOn200() {
+        $mock = new MockHandler([
+            new Response(200, ["SOME CONTENT"]),
```

Comment:
You'll need to setup a fake response with headers ala https://wiki.duraspace.org/display/FEDORA40/RESTful+HTTP+API+-+Containers#RESTfulHTTPAPI-Containers-GreenHEADtrueRetrievetheresourceheaders .  Then you'll have to check for them further down.  Wouldn't need to be exhaustive, just pick a few like Accept-Patch and Accept-Post.